tests/alert-max/alert-max-append-higher-priority/suricata.yaml

```diff
@@ -0,0 +1,21 @@
+%YAML 1.1
+---
+
+outputs:
+  - stats:
+      enabled: yes
+      filename: stats.log
+      append: yes       # append to file (yes) or overwrite it (no)
+      totals: yes       # stats for all threads merged together
+      threads: no       # per thread stats
+  - eve-log:
+      enabled: yes
+      filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
+      filename: eve.json
+      types:
+        - alert
+        - http
+
+# Define maximum number of possible alerts that can be triggered for the same
+# packet. Default is 15
+packet-alert-max: 2
```

Comment:
packet-alert-max had to be `2` instead of the otherwise obvious 3, because some of the rules are inspected against a fake packet, apparently, leading to a packet-alert-max value = 3 allowing for 4 alerts to be queued.